test/Interop/Cxx/objc-correctness/Inputs/NSOptionsMangling.h

```diff
@@ -0,0 +1,50 @@
+#define __CF_OPTIONS_ATTRIBUTES __attribute__((flag_enum,enum_extensibility(open)))
```

Comment:
In general, I think this test can be a bit more concise.

Reply:
I will reduce the test header soon.